static/app/views/setupWizard/index.tsx

```diff
@@ -1,4 +1,5 @@
 import {useCallback, useEffect, useMemo, useRef, useState} from 'react';
+import {createBrowserRouter, RouterProvider} from 'react-router-dom';
```

Comment:
I wish we didn't have all these one off react rendering places, because as we add more global things trying to maintain these places becomes harder and harder